docs/content/compress.md

```diff
@@ -0,0 +1,136 @@
+---
+title: "Compress"
+description: "Compression Remote"
+date: "2020-09-14"
+---
+
+Compress (Experimental)
+-----------------------------------------
+
+The `Compress` remote adds compression to another remote. It is best used with remotes containing
+many large compressible files.
+
+Please read the [warnings](#warnings) before using this remote.
+
+To use this remote, all you need to do is specify another remote and a compression mode to use:
+
+```
+Current remotes:
+
+Name                 Type
+====                 ====
+remote_to_press      sometype
+
+e) Edit existing remote
+$ rclone config
+n) New remote
+d) Delete remote
+r) Rename remote
+c) Copy remote
+s) Set configuration password
+q) Quit config
+e/n/d/r/c/s/q> n
+name> compress
+...
+ 8 / Compress a remote
+   \ "compress"
+...
+Storage> compress
+** See help for compress backend at: https://rclone.org/compress/ **
+
+Remote to compress.
+Enter a string value. Press Enter for the default ("").
+remote> remote_to_press:subdir 
+Compression mode.
+Enter a string value. Press Enter for the default ("gzip").
+Choose a number from below, or type in your own value
+ 1 / Gzip compression balanced for speed and compression strength.
+   \ "gzip"
+compression_mode> gzip
+Edit advanced config? (y/n)
+y) Yes
+n) No (default)
+y/n> n
+Remote config
+--------------------
+[compress]
+type = compress
+remote = remote_to_press:subdir
+compression_mode = gzip
+--------------------
+y) Yes this is OK (default)
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+### Compression Modes
+Currently only gzip compression is supported, it provides a decent balance between speed and strength and is well
+supported by other application. Compression strength can further be configured via an advanced setting where 0 is no
+compression and 9 is strongest compression.
+
+#### Filetype
+If you open a remote wrapped by press, you will see that there are many files with an extension corresponding to
+the compression algorithm you chose. These files are standard files that can be opened by various archive programs, 
+but they have some hidden metadata that allows them to be used by rclone.
+While you may download and decompress these files at will, do **not** upload any compressed files to a wrapped remote
+through any other means than rclone. This will upload files that do not contain metadata and **will** cause unexpected behavior.
```

Comment:
What happens? :-)

Reply:
If I didn't break anything they should in fact just be ignored. (I haven't tested this in a long time though)